README.md

```diff
@@ -10,3 +10,22 @@
 - Free software: GNU Affero General Public License v3+
 
 ## Coming Soon™
+
+
+## Get started ##
+
+We use [poetry](https://python-poetry.org/) for packaging and dependency management,
```

Comment:
The project's `pyproject.toml` makes it [PEP-517](https://peps.python.org/pep-0517/) by default, but the package is also [published on PyPI](https://pypi.org/project/celery-aio-pool/).

Should we specify that it's directly installable via `pip install celery-aio-pool` if a user doesn't explicitly use (or want to use) Poetry? Also, shouldn't we just refer to [Poetry's installation documentation](https://python-poetry.org/docs/#installation) (so we don't have to include installation instructions for _all_ supported platforms)?